### PR TITLE
fix: Improve accessibility for OPF payments

### PR DESCRIPTION
**Pull request by rmch91**
Closes:
https://jira.tools.sap/browse/CXSPA-8928
https://jira.tools.sap/browse/CXSPA-8927

Check with explicit and not terms and conditions

### CODE REVIEW

**Review by developpeurweb — DISMISSED** [review]

**Review by FollowTheFlo — DISMISSED** [review]
Looks good to me.
tested with and without ExplicitT&C
tested with several customized payment option info label
tested build:libs

File: integration-libs/opf/base/root/config/opf-config.ts [hunk 1]
```diff
@@ -15,6 +15,26 @@ export abstract class OpfConfig {
...
+       *     }
+       *   }
+       * };
+       * ```
+       */
+      paymentInfoMessagesMap?: Record<number, string>;
+      /**
+       * Enables the payment info message section within the payment options
+       */
+      enableInfoMessage?: boolean;
```
Comment (FollowTheFlo) [inline integration-libs/opf/base/root/config/opf-config.ts:36]: this flag value is not read anywhere, shouldn't be it use in template or component on condition to display the InfoMessage?

Comment (rmch91) [reply integration-libs/opf/base/root/config/opf-config.ts:36]: changed

File: integration-libs/opf/checkout/components/opf-checkout-payments/opf-checkout-payments.component.ts [hunk 2]
```diff
@@ -89,6 +96,21 @@ export class OpfCheckoutPaymentsComponent implements OnInit, OnDestroy {
...
+    const translationKey =
+      paymentId && this.opfConfig?.opf?.paymentOption?.paymentInfoMessagesMap
+        ? (this.opfConfig.opf.paymentOption.paymentInfoMessagesMap[paymentId] ??
+          defaultMessage)
+        : defaultMessage;
+
+    return this.translation.translate(translationKey);
+  }
+
+  get ifPaymentInfoMessageVisible(): boolean {
```
Comment (FollowTheFlo) [inline integration-libs/opf/checkout/components/opf-checkout-payments/opf-checkout-payments.component.ts]: typo:
ifPaymentInfoMessageVisible
to
isPaymentInfoMessageVisible

Comment (rmch91) [reply integration-libs/opf/checkout/components/opf-checkout-payments/opf-checkout-payments.component.ts]: changed

**Review by FollowTheFlo — APPROVED** [review]
good to me as discussed.

**Pull request merged**